app/code/Magento/Eav/Setup/EavSetup.php

```diff
@@ -930,13 +930,16 @@ public function addAttributeOption($option)
             }
         } elseif (isset($option['values'])) {
             foreach ($option['values'] as $sortOrder => $label) {
+$checkOpionId = $this->setup->getConnection()->select($optionTable, ['attribute_id =?' =>$option['attribute_id'], 'sort_order =?' => $sortOrder, 'value =?'=> $label]);
```

Comment:
Please consider moving this select out of the loop if possible

Reply:
we have to check each option before insert that's I have put select inside loop.
Any suggestion?